config.go

```diff
@@ -68,12 +70,14 @@ func LoadConfig(configFile string) (*Config, error) {
 		MsgOnce:         false,
 		UsePrivmsg:      false,
 		AlertBufferSize: 2048,
+		NickservName:    "NickServ",
```

Comment:
Can you please add NickservName and ChanservName to the config used in the tests? See https://github.com/google/alertmanager-irc-relay/blob/master/irc_test.go#L41